src/taipy/gui/gui.py

```diff
@@ -1636,6 +1635,7 @@ def _hold_actions(
         message: t.Optional[str] = "Work in Progress...",
     ):  # pragma: no cover
         action_name = callback.__name__ if callable(callback) else callback
+        # TODO: what if lambda? (it does work)
```

Comment:
it does work as long as there is only one declaration (the other lambda are named by their code :-)

Reply:
True.